docker/BTC/docker-compose.yml

```diff
@@ -23,39 +23,86 @@ services:
   indexer:
     container_name: btc-indexer
     image: blockcore/indexer:0.2.32
+    deploy:
+        resources:
+            limits:
+              cpus: 0.200
+              memory: 256M
     environment:
       VIRTUAL_HOST: btc.indexer.blockcore.net
       VIRTUAL_PORT: 9910
+      VIRTUAL_PROTO: http
       VIRTUAL_NETWORK: proxy
       LETSENCRYPT_HOST: btc.indexer.blockcore.net
       LETSENCRYPT_EMAIL: admin@blockcore.net
+      ASPNETCORE_URLS: http://+:9910
     command: ["--chain=BTC"]
     restart: unless-stopped
     depends_on:
-        - mongo
-        - chain
+      mongo:
+        condition: service_healthy
+      chain:
+        condition: service_healthy
+    networks:
+      - btcnetwork
```

Comment:
Does this remove the need for manually connecting networks after creation? That would be great =)

Reply:
This is to specify which networks the service is able to access.

The section below allows you to specify networks to automatically create.
No need to create the network manually.

https://github.com/block-core/chaininfo/blob/57aea404812c4437245915dabfb0ddfe3c9aa052/docker/BTC/docker-compose.yml#L99

Reply:
Yes. I need to manually connect to the network.